### PR TITLE
feat: add typefully card to open page

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

The new card for typefully looks like the below screenshot

Screenshot 2024-03-01 at 12 53 00

### CODE REVIEW

**Review by catalinpit — COMMENTED** [review]
The white background makes it hard to see the month. The green color on white is also not pleasant.

issue

Design breaks on smaller screen.
issue2

**Comment by adithyaakrishna** [comment]
@catalinpit Made the requested changes :)

**Comment by Mythie** [comment]
Consider using `import { FaXTwitter } from 'react-icons/fa6';` rather than whatever image is currently being used. It'll look nicer and be themeable across dark and light modes.

File: apps/marketing/src/app/(marketing)/open/monthly-new-users-chart.tsx [hunk 1]
```diff
@@ -32,6 +32,9 @@ export const MonthlyNewUsersChart = ({ className, data }: MonthlyNewUsersChartPr
             <YAxis />
 
             <Tooltip
+              labelStyle={{
+                color: 'hsl(var(--primary-foreground))',
+              }}
```
Comment (Mythie) [inline apps/marketing/src/app/(marketing)/open/monthly-new-users-chart.tsx:37]: Why was this changed as part of this PR?

File: apps/marketing/src/app/(marketing)/open/monthly-total-users-chart.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export type MonthlyTotalUsersChartProps = {
 export const MonthlyTotalUsersChart = ({ className, data }: MonthlyTotalUsersChartProps) => {
   const formattedData = [...data].reverse().map(({ month, cume_count: count }) => {
     return {
-      month: DateTime.fromFormat(month, 'yyyy-MM').toFormat('LLL'),
+      month: DateTime.fromFormat(month, 'yyyy-MM').toFormat('LLLL'),
```
Comment (Mythie) [inline apps/marketing/src/app/(marketing)/open/monthly-total-users-chart.tsx:17]: Same ting for all of these

File: apps/marketing/src/app/(marketing)/open/typefully.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import { Button } from '@documenso/ui/primitives/button';
+
+export type TypefullyProps = HTMLAttributes<HTMLDivElement>;
+
+export const Typefully = ({ className, ...props }: TypefullyProps) => {
+  const [isSSR, setIsSSR] = useState(true);
+
+  useEffect(() => {
+    setIsSSR(false);
+  }, []);
```
Comment (Mythie) [inline apps/marketing/src/app/(marketing)/open/typefully.tsx:20]: We have a mounted hook for this but I'm not sure as to why you would need it for the following code.

File: apps/marketing/src/app/(marketing)/open/monthly-new-users-chart.tsx (hunk 1 above)
Comment (adithyaakrishna) [reply apps/marketing/src/app/(marketing)/open/monthly-new-users-chart.tsx:37]: The label colors weren't visible currently, so updated it to match the others

[Typefully.webm](https://github.com/documenso/documenso/assets/23498248/2ea2de1a-695d-49f3-8b0e-302d4d18dc6f)

File: apps/marketing/src/app/(marketing)/open/monthly-total-users-chart.tsx (hunk 2 above)
Comment (adithyaakrishna) [reply apps/marketing/src/app/(marketing)/open/monthly-total-users-chart.tsx:17]: I thought instead of just showing it as Mar Feb, why don't we show it completely as March or February 😶

File: apps/marketing/src/app/(marketing)/open/monthly-new-users-chart.tsx (hunk 1 above)
Comment (adithyaakrishna) [reply apps/marketing/src/app/(marketing)/open/monthly-new-users-chart.tsx:37]: After the label changes, they look more like this, 

Screenshot 2024-03-05 at 15 31 19

File: apps/marketing/src/app/(marketing)/open/typefully.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+export type TypefullyProps = HTMLAttributes<HTMLDivElement>;
+
+export const Typefully = ({ className, ...props }: TypefullyProps) => {
+  return (
+    <div className={cn('flex flex-col', className)} {...props}>
+      <h3 className="px-4 text-lg font-semibold">Twitter Stats</h3>
+
+      <div className="border-border mt-2.5 flex flex-1 items-center justify-center rounded-2xl border py-8 shadow-sm hover:shadow">
+        <div className="flex flex-col items-center gap-y-4 text-center">
+          <FaXTwitter className="h-12 w-12" />
```
Comment (Mythie) [inline apps/marketing/src/app/(marketing)/open/typefully.tsx:21]: I assume that this will need some `text-foreground` css class to make sure it inverts as needed for light/dark mode? 🤔

**Review by Mythie — APPROVED** [review]

File: apps/marketing/src/app/(marketing)/open/typefully.tsx (hunk 4 above)
Comment (adithyaakrishna) [reply apps/marketing/src/app/(marketing)/open/typefully.tsx:21]: I dont think so, the logo didn't need any other CSS class apart from the sizing ones and the themes changed correctly 😶 
[Twitter-Logo.webm](https://github.com/documenso/documenso/assets/23498248/6376582b-0771-40ca-939e-155eb48906b8)
